### PR TITLE
[Task] add user model #TDB-104

### PR DESCRIPTION
**Pull request by anjakammer**
- [x] user model
- [x] user model test
- [x] association with m-users <-> m-projects 
- [x] association tests for users and projects
- [x] mail address should be unique
- [x] user can be added with param :
```
{
user_ids : [1, 2]
}
```
- [x] validation, filtering
- [x] api test

### CODE REVIEW

**Comment by ghost** [comment]
@anjakammer I'm thinking the email should probably be unique, no?

**Comment by anjakammer** [comment]
@mvuajua - you are right - I will fix that. The rest is fine?

**Comment by ghost** [comment]
@anjakammer yes!

**Comment by anjakammer** [comment]
@mvuajua all set - please take a look again. :)

File: true [hunk 1]
```diff
@@ -0,0 +1,5 @@
+[2K[1G[1myarn check v1.3.2[22m
```
Comment (ghost) [inline true]: the fuck is this file :joy:

Comment (anjakammer) [reply true]: Ähhh don't know :D

**Review by ghost — APPROVED** [review]

**Pull request merged**